changelog/fix-4508-remove-settings-submenu-ufe-notification-badge

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Remove the UFE notification badge on the Payments > Settings menu item
```

Comment:
Small typo here, this should be `UPE`, not `UFE`. 🙂 (It stands for Universal Payment Element)

Reply:
Thank you @dmallory42 for reviewing this! Good catch on the typo - fixed it!